internal/olm/operator/registry/configmap/deployment.go

```diff
@@ -25,10 +25,9 @@ import (
 )
```

Comment:
Missing changelog: (see [changelog/fragments/00-template.yaml](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))

Reply:
Is that a change? Why we are changing it?
Is it to fix a bug? Why do we need to replace image A for B?